docs/conditional.rst

```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
         views = NumberAttribute(default=0)
 
 
-AND vs. OR
-^^^^^^^^^^
+Condition Expressions
```

Comment:
I think it'd be really useful to show the conditions being used in a real query, at least once, here